rust/examples/config.ini

```diff
@@ -0,0 +1,21 @@
+[station]
+id="ora_opensource"
```

Comment:
`ora_copycat` ?

Reply:
I'll switch
I still need to go again through the whole configuration/monitoring/etc. to re-understand what this id is used for compared to the client ID, where it is used and for what exactly
It is mandatory now, but mobility related so changes will happen due to IoT isolation